app/jobs/discord_messages/visibility_job.rb

```diff
@@ -0,0 +1,18 @@
+module DiscordMessages
+  class VisibilityJob < ApplicationJob
+    queue_as :urgent
+
+    def perform(message_id, resolver_name)
+      @message = Message.find(message_id)
+      return if message.expired?
+
+      Notifications::DiscordNotifier.new(message).notify_message_state!(resolver_name)
+    rescue StandardError => e
+      Rails.logger.error("Error at DiscordMessages::VisibilityJob: #{e.message}")
+    end
```

Comment:
same here. Why rescue this?
```suggestion
    end
```

Reply:
yeah, I agree, done!